src/styles/stars.css

```
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  top: 0;
+  height: 16px;
+  width: 16px;
+  background: url(../images/star.svg) no-repeat center;
+  border-radius: 1px;
+  background-repeat: no-repeat;
+}
+
+.star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .star:nth-child(-n + 1) {
+  background: url(../images/star-active.svg) no-repeat center;
+}
+
+.stars--2 .star:nth-child(-n + 2) {
+  background: url(../images/star-active.svg) no-repeat center;
+}
```

```suggestion
.stars--1 .star:nth-child(-n + 1),
.stars--2 .star:nth-child(-n + 2) {
  background: url(../images/star-active.svg) no-repeat center;
}
```
combine all those selectors with comma